src/main/java/seedu/address/model/hotel/room/Tier.java

```diff
@@ -1,10 +1,11 @@
-package seedu.address.model.hotel;
+package seedu.address.model.hotel.room;
 
 
 /**
  * Store room tier.
  */
 public class Tier {
+    public static final String MESSAGE_CONSTRAINTS = "Tier must be gold, silver or bronze";
```

Comment:
Could we put the tier in ENUM? To make it flexible.

Reply:
If there are options to add tier, then better to maintain a unique tier list. However, I see no problem with keeping tier name as string.

Reply:
I think we can definitely keep this in mind to work on later.

Reply:
Or list of string is actually also fine. Yesterday the problem is it manually written bronze, silver and gold which will be really difficult if we want to change the tiers option. But enum works just fine as well.